README.md

```diff
@@ -394,7 +394,7 @@ The back-end uses the [Django translation framework][] for internationalization.
 To extract messages for localization, run:
 
 ```
-python manage.py makemessages -e py
+yarn django:makemessages
```

Comment:
Bleh, we have to pass so many weird flags to Django's `makemessages` command that I found it easier to just create a Yarn command that did it for us.